portfolio/src/main/webapp/css/images.css

```diff
@@ -0,0 +1,33 @@
+.image-container {
+    display: flex;
```

Comment:
nit: IIRC at Google all attributes are supposed to be sorted alphabetically.

Reply:
Good idea, I will go change them

Reply:
Done